src/mvp/objects/NetworkManager.cpp

```diff
@@ -158,42 +158,47 @@ void NetworkManager::GameStop() {
 }
 
 void NetworkManager::UpdateBoard(Message &msg) {
-    uint8_t player1Ships;
-    uint8_t player2Ships;
-    msg >> player1Ships >> player2Ships;
+    uint32_t playerTopId;
+    uint32_t playerBottomId;
+    msg >> playerBottomId >> playerTopId;
```

Comment:
I feel like this is not the way to handle this. In my PR, this is sent once and stored. I guess we can leave it in now and I can remove it later when integrating